.lycheeignore

```diff
@@ -0,0 +1,40 @@
+# .lycheeignore: URLs and patterns to skip during lychee link checking
```

Comment:
👍 

Maybe check with what we have on `main` too:
https://github.com/kedro-org/kedro/blob/main/docs/source/conf.py#L243